pjsip/include/pjsip/sip_transaction.h

```diff
@@ -98,7 +98,9 @@ struct pjsip_transaction
     pjsip_method                method;         /**< The method.            */
     pj_int32_t                  cseq;           /**< The CSeq               */
     pj_str_t                    transaction_key;/**< Hash table key.        */
+    pj_str_t                    transaction_key2;/**< Hash table key (2).   */
     pj_uint32_t                 hashed_key;     /**< Key's hashed value.    */
+    pj_uint32_t                 hashed_key2;    /**< Key's hashed value (2).*/
```

Comment:
I think it should be clear from the name, for what it is used. (instead of hashed_key2)

Reply:
Or perhaps adding some hints that it is part of merged request handling in the comment/doc is IMHO sufficient, to avoid too long var/field name.

Reply:
I did consider this for a while, with the only other candidate I can think of is `*_uas`, i.e. `htable_uas, `hashed_key_uas`, etc.

Despite its simplicity, I thought `htable2` is the best option since it indicates there are two hash tables, obviously with different keys. And the transaction lookup flow is done by comparing to the main hash table, and then to the secondary hash table. So `htable2` for *second*ary hash table seems pretty intuitive.

I'm open to other suggestions though. For now, I just add more comment in the doc.